examples/encryption/main.tf

```diff
@@ -0,0 +1,24 @@
+
```

Comment:
Can you change this path from `examples/encryption` to `examples/ s3-bucket-key` to make it clear what the example is doing?

Reply:
Done! Thanks for taking a look.